add border setting

Iksas opened the pull request:
This PR implements an additional setting that adds a thin black border to the "scanned" page.

The added button:
![toggle](https://user-images.githubusercontent.com/17005217/164223266-8925fff1-3dc4-4894-8ef6-23be1f425089.png)

The result:
![result](https://user-images.githubusercontent.com/17005217/164223285-8856f427-4527-4447-ac8b-d7ad0f63bdd8.png)

rwv commented:
Maybe default border config should be false? It is this kind of artifact should be opt-in instead of opt-out.

Iksas reviewed (commented):
You’re right, default behavior shouldn’t be changed 👍

rwv commented:
Fix border check https://github.com/rwv/lookscanned.io/commit/71b1813727ee6233b20b1a1294e81beaddb87e0e

rwv commented:
@Iksas Could you explain why border need be called twice with black border and white border? I don't understand it.

Iksas commented:
@rwv Every time I tried it with a single border, black triangles would appear on the edge of the page. I assumed that this was caused by some quirk of Imagemagick, and implemented a workaround with two borders. 

After some investigation today, I found out that I was wrong. The triangles were actually caused by a bug I introduced in #3. I submitted a fix in #7.
